compiler/luci/pass/src/helpers/LayerInfoMap.cpp

```diff
@@ -56,8 +198,6 @@ LayerInfoMap layer_info_map(loco::Graph *g, std::vector<LayerInfo> &layers_info)
                                ". Check layer names in the quantization configuration file.");
   }
 
-  assert(info_by_name.size() == layers_info.size()); // FIX_ME_UNLESS
-
```

Comment:
why is this remove?

Reply:
Function:
```c++
void add_multi_output_node(LayerInfoMap &info_by_name, LayerInfo &layer_info,
                           luci::CircleNode *node)
```
adds to `info_by_name` all outputs nodes (of multiple output node), so the size of the `info_be_name` can be larger than `layers_info`.
here I tried to explain the reason for this addition: https://github.com/Samsung/ONE/pull/8673#discussion_r829793911